### DISCUSSION TITLE
Restore all instances after update

### DISCUSSION THREAD

**Discussion by bcardarella** [zed-industries/zed#11924] [Feature Requests]
When prompted to update and restart Zed only the currently focused instance of Zed is restored. It would be nice if the restart for update restored all instances of Zed that were open.

**Comment by SomeoneToIgnore** [comment]
https://github.com/zed-industries/zed/issues/4985 and https://github.com/zed-industries/zed/issues/9851 track this, and I agree that it would be very nice to have.
